_templates/layout.html

```diff
@@ -5,14 +5,19 @@
 <!--  MathJax (LaTex math) -->
 <script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML" data-aplus></script>
 
+<!-- Annotated text -->
+<script type="text/javascript"
```

Comment:
I think that the JS and CSS for the annotated code block should be added automatically by the a-plus-rst-tools. The course would not need to add the static files then.

Reply:
I will make the changes once the Aalto-LeTech/a-plus-rst-tools#82 is merged